modules/openy_gc_log/js/logger.behaviors.js

```diff
@@ -0,0 +1,15 @@
+(function ($, Drupal) {
```

Comment:
Why we need jquery for this? Can we make this api call with axios from the VUEX action?